samples/nrf9160/lwm2m_client/src/ui/include/ui_buzzer.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef UI_BUZZER_H__
+#define UI_BUZZER_H__
+
+#include <zephyr.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Turn the buzzer on or off. If false the buzzer
+ * will be off regardless of the frequency and the intensity.
```

Comment:
Genral comment, The docs are a bit easier to read if you align the indentation within paragraph like this, it's what we do typically in other NCS headers.
```suggestion
 * @brief Turn the buzzer on or off. If false the buzzer
 *        will be off regardless of the frequency and the intensity.
```

Reply:
Fixed